packages/now-node/src/index.ts

```diff
@@ -23,10 +25,28 @@ interface DownloadOptions {
   files: Files;
   entrypoint: string;
   workPath: string;
-  meta?: Meta;
+  meta: Meta;
   npmArguments?: string[];
 }
 
+const watchers: Map<string, NccWatcher> = new Map();
+
+function getWatcher(entrypoint: string, options: NccOptions): NccWatcher {
+  let watcher: NccWatcher | void = watchers.get(entrypoint);
```

Comment:
```suggestion
  let watcher = watchers.get(entrypoint);
```

The type is redundant